WordPress/Classes/ViewRelated/Blog/SharingButtonsViewController.swift

```diff
@@ -843,6 +819,16 @@ import WordPressShared
     }
 
 
+    override func tableView(tableView: UITableView, canEditRowAtIndexPath indexPath: NSIndexPath) -> Bool {
+        // We only actual editing (ordering) with sections that canSort and are not toggled in the UI activating/deactivating.
+        let section = sections[indexPath.section]
+        if section.canSort && !section.editing && indexPath.row > 0 {
```

Comment:
Could you just `return (section.canSort && !section.editing && indexPath.row > 0)`?